tests/test_decoders.py

```diff
@@ -128,18 +129,21 @@ def test_decoding_errors(header_value):
         ),
     ],
 )
-def test_text_decoder(data, encoding):
-    def iterator():
+@pytest.mark.asyncio
+async def test_text_decoder(data, encoding):
+    async def iterator():
         nonlocal data
         for chunk in data:
             yield chunk
 
     response = httpx.Response(200, content=iterator())
-    assert "".join(response.stream_text()) == (b"".join(data)).decode(encoding)
+    await response.read()
+    assert "".join(response.text) == (b"".join(data)).decode(encoding)
```

Comment:
While this should *work*, we can simplify:

```suggestion
    assert response.text == (b"".join(data)).decode(encoding)
```